src/main/java/pivotaltracker/ui/Permalink.java

```diff
@@ -23,7 +23,7 @@
  */
 public final class Permalink {
     public static final String LOGIN_PAGE = "/signin";
-    public static final String DASHBOARD_PAGE = "/dashboard";
+    public static final String DASHBOARD_PAGE = "/workspace";
```

Comment:
why you page workspace called DASHBOARD_PAGE ?

Reply:
solve